.env.sample

```diff
@@ -1,9 +0,0 @@
-PORT=8000
-DS_API_URL=http://localhost:8002
-DS_API_TOKEN=SUPERSECRET
-DEV_DATABASE_URL=postgres://docker:docker@127.0.0.1:5400/api-dev
```

Comment:
We likely want to remove the 2nd "docker" in line 4 and 5, but I think the instructions for setup reference this file as an example. Probably not a good idea to remove the file altogether.